src/subsai/cli.py

```diff
@@ -53,6 +53,8 @@ def _handle_media_file(media_file_arg: list[str]):
 
 
 def _handle_model_configs(model_configs_arg: str):
+    if type (model_configs_arg) != str:
+        return model_configs_arg
```

Comment:
The function should always be accepting an `str` object as input, (what `argparse` is  returning actually) 
so I think adding a test about the type inside the function is not the way to go in my opinion. 
I think just changing the default of the `model_configs` in the argparse commands from raw dict object to `str` `'{}'` will resolve the issue. 

What do you think ?

Reply:
If the other parts of the program can handle `"{}"` instead of `{} ` that is also fine. I was not sure about that.

Reply:
I think it will be handled correctly.
Please give it a try on your end, if it works just submit another commit and I will merge it.

Reply:
> I think it will be handled correctly. Please give it a try on your end, if it works just submit another commit and I will merge it.

Changed, tested and committed.

Reply:
Perfect, thanks.